From 77802b2f63d35f47890e9246d34b9c3c7b3bca70 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Tue, 16 Aug 2022 09:45:22 +0200
Subject: [PATCH 01/52] vduse: Remove include of rwlock.h
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/6.1/older/patches-6.1.19-rt8.tar.xz

rwlock.h should not be included directly. Instead linux/splinlock.h
should be included. Including it directly will break the RT build.

Remove the rwlock.h include.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Acked-by: Michael S. Tsirkin <mst@redhat.com>
Link: https://lkml.kernel.org/r/20221026134407.711768-1-bigeasy@linutronix.de
(cherry picked from commit 10745848677412d239701f16b459e53d2ed83119)
Signed-off-by: Clark Williams <clark.williams@gmail.com>
---
 drivers/vdpa/vdpa_user/iova_domain.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/vdpa/vdpa_user/iova_domain.h b/drivers/vdpa/vdpa_user/iova_domain.h
index 4e0e50e7ac15..173e979b84a9 100644
--- a/drivers/vdpa/vdpa_user/iova_domain.h
+++ b/drivers/vdpa/vdpa_user/iova_domain.h
@@ -14,7 +14,6 @@
 #include <linux/iova.h>
 #include <linux/dma-mapping.h>
 #include <linux/vhost_iotlb.h>
-#include <linux/rwlock.h>
 
 #define IOVA_START_PFN 1
 
-- 
2.39.2

